### DISCUSSION TITLE
docstring for Pluto cells

### DISCUSSION THREAD

**Discussion by mind6** [fonsp/Pluto.jl#2354] [Pluto development] [answered]
I would like to see the ability to write a docstring for any individual cell. Sometimes a cell is there to provide state, I don't want to see the printout of the state, just the docstring for what that state represents. This way hidden cells can show a meaningful output instead of occupying a darkened area on the notebook page. 

Right now if you put a string on top of a begin/end block, Pluto actually tries to process it as a docstring but fails to do so.
```
cannot document the following expression:

begin
<julia code>
end

error(::String, ::String)@error.jl:44
top-level scope@[Local: 1](http://localhost:1234/edit?id=ae68b340-5c36-11ed-3b95-6de589f0a234#)[inlined]
```

**Comment by disberd** [comment, marked as answer]
Hi, Pluto just outputs the last expression in the block to the frontend.

From what I understand from what you write, you just want to have the output provide some information on what was done in the begin-end block so that you can hide the input and just see a meaningufl comment.

If this is the case, and especially if you are using a `begin .. end` block, you can just put your description string as the last instruction before the `end`. The remaining code will still be executed in global scope but just the string will be sent to the output.
